src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0, // result is a property of the calculator object
+
+    // Define the basic operations
+    add: function (num) {
+      this.result += num;
+    },
```

Comment:
[CODE KNOWLEDGE] You can use a shorter syntax for the methods in your object. Instead of `add: function (num) {...}` you can write `add(num) {...}`.